Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -59,7 +59,8 @@ class TopSiteItemCell: UICollectionViewCell {
 
     override init(frame: CGRect) {
         super.init(frame: frame)
-
+        isAccessibilityElement = true
+        accessibilityLabel = "TopSite"
```

Comment:
hmm it seems like with the new GH review. Outdated comments aren't collapsed. Probably better to use the new review process instead.